plugins/iexec.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: iexec
+spec:
+  version: "0.1.9"
```

Comment:
this is not matching the url below.
can you also start this string with `v`?

Reply:
Fixed & updated to the latest version